backend/src/main/java/capstone/backend/model/db/order/OrderToSupplier.java

```diff
@@ -7,16 +7,18 @@
 import javax.persistence.Entity;
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
+import java.util.List;
 
 @ToString
-@AllArgsConstructor
 @Entity
 @SuperBuilder
+@NoArgsConstructor
 @Table(name = "supplier_orders")
 public class OrderToSupplier extends Order {
 
-    public OrderToSupplier() {
-        super();
+    public OrderToSupplier(Long id, List<OrderItem> orderItems, Supplier supplier) {
```

Comment:
when you work with Lombok you can delete constructor and add the @requiredArgsConstructor

Reply:
That only works if the fields are final (non final fields are not required). But the fields cannot be final because JPA needs a default constructor.

Constructors should be placed between attributes and methods. See https://www.oracle.com/technetwork/java/codeconventions-150003.pdf